css/bootstrap.scss

```diff
@@ -869,6 +871,42 @@ body.theme-purple {
     }
   }
 
+  // add `before` to code table entries to account for top navbar in anchors
+  dl.class,
+  dl.function,
+  dl.attribute,
+  dl.method {
+    > dt::before {
+      content: "";
+      display: block;
+      height: 6rem;
```

Comment:
Where does the 6 come from? Should this be made a variable?

Reply:
Added a fixup switching this to a variable. It turns out we already had a variable for the header height. It was specified in absolute pixels rather than rem, but I figured it made sense to use the same value that we were using elsewhere for the header height, so I switched to that.